### DISCUSSION TITLE
GitHub Desk Desktop

### DISCUSSION THREAD

**Discussion by IGotADreamInMind** [community/community#24276] [Mobile]
<p>Hey, folks how is everyone?<br>
Thanksgsgiving is right around the corner. So I hope everyone that we’ve all choosen to has a good one! Can GitHub Desktop be downloaded on android Samsang Galaxy Note?</p>

**Comment by wadleo** [comment]
<p>Hi <a class="mention" href="/igotadreaminmind">@igotadreaminmind</a>,</p>
<p>I am not really sure of what you are trying to ask but if I understand your question, you wish to know if Github for Desktop is installable on an android device. Github desktop is for a desktop pc like a Mac or Pc. Not for mobile. So you can install it on your Samsung Galaxy Note. Have a great day and enjoy your Thanksgiving holidays.</p>

**Comment by IGotADreamInMind** [comment, marked as answer]
<p>Ok thank you. Appreciate your answer <a class="mention" href="/wadleo">@wadleo</a>.</p>

**Comment by wadleo** [comment]
<p>You are welcome and have a great thanksgiving.</p>
